README.md

```diff
@@ -0,0 +1,73 @@
+# XHarness
+
+This repo contains the code to build the **XHarness dotnet tool**.
+
+## What is XHarness
+
+XHarness is primarily a command line tool that enables running tests on Android and iOS (WatchOS and tvOS are also supported).
+It can locate devices/emulators, install given application, run it and collect results (uninstall it after).
+It handles crashes gracefully and supports different types of connection modes (Network, USB cable).
```

Comment:
Gracefully is a subjective term and doesn't mean anything.  If you're going to say how it handles crashes it should be more explicit, like "Handles bad machine state or crashing APKs by detecting this, logging as much as possible ... yada"